x/records/migrations/v2/convert.go

```diff
@@ -43,7 +43,6 @@ func convertToNewEpochUnbondingRecord(oldEpochUnbondingRecord oldrecordstypes.Ep
 func convertToNewUserRedemptionRecord(oldRedemptionRecord oldrecordstypes.UserRedemptionRecord) recordstypes.UserRedemptionRecord {
 	return recordstypes.UserRedemptionRecord{
 		Id:             oldRedemptionRecord.Id,
-		Sender:         oldRedemptionRecord.Sender,
```

Comment:
yeah the old migration code doesn't really matter haha, it's more just a source of documentation

I removed it here cause otherwise it wouldn't be possible to compile because the type no longer has the sender field

As a rule of thumb, I think we should try to avoid changing these as much as possible, but I don't think small things like this matter much!

Reply:
@shellvish can I mark this resolved?

Reply:
maybe just comment it out (nit)

Reply:
good idea, done!